pkg/api/2018-09-30-preview/default.go

```diff
@@ -0,0 +1,13 @@
+package v20180930preview
+
+func setDefaults(oc *OpenShiftManagedCluster) {
+	defaultRouterName := "default"
+
+	if len(oc.Properties.RouterProfiles) == 0 {
+		oc.Properties.RouterProfiles = []RouterProfile{
+			{
+				Name: &defaultRouterName,
```

Comment:
in our codebase prefer using to.StringPtr() to skip the extra variable definition